### ISSUE TITLE
Content not announced while VoiceOver on

### ISSUE THREAD

**Issue by qanitah** [CovidShield/mobile#161]
While VoiceOver is on, some content is not announced/skipped when the user is reading the full page (highlighted in yellow in screenshot). 

- it doesn't skips "Francais" on first tab and "Back" on others
- on 1st tab, it doesn't say "Next" and goes to tab 2; same for 3rd tab and goes to 4th tab
- it doesn't say tab # (at the bottom)

![stopcovid1](https://user-images.githubusercontent.com/14280062/85785659-843d2b00-b6f7-11ea-8d66-88c664e85451.jpg)
